astropy/convolution/convolve.py

```diff
@@ -160,7 +160,7 @@ def convolve(array, kernel, boundary='fill', fill_value=0.,
 
     Parameters
     ----------
-    array : `~astropy.nddata.NDData` or `numpy.ndarray` or array_like
+    array : `~astropy.nddata.NDData` or ndarray or array-like
```

Comment:
replace first "or" with ","

Reply:
Changing instead to `` `~astropy.nddata.NDData` or array-like``

Reply:
✅